src/main/java/step5/RacingGame.java

```diff
@@ -0,0 +1,42 @@
+package step5;
+
+import step5.View.InputView;
+import step5.domain.Cars;
+import step5.domain.RandomGenerate;
+import static step5.View.OutputView.*;
+
+public class RacingGame {
+
+    private final Cars cars;
+    private int attempt;
+
+    public RacingGame(InputView inputView) {
+        String carNames = inputView.inputCarNames();
+        this.attempt = inputView.inputAttempt();
+        this.cars = new Cars(carNames);
+    }
+
+    public void start() {
+        int round = attempt;
+        printStart();
+        while (round-- != 0) {
+            move();
+            printCarMove(cars);
+
+        }
+        printWinner();
+
+    }
+
+    public void move() {
+        attemptRacing();
+    }
+
+    private void attemptRacing() {
+        cars.getCars().forEach(
+                i -> i.move(new RandomGenerate())
```

Comment:
움직일때마다 RandomGenerate 인스턴스가 생성되는 구조네요 😭 
RandomGenerate 인스턴스를 한번만 생성되어 재사용할 수 있도록 변경해주시면 어떨까요 :) ?
```suggestion
                car-> car.move(generateNumber)
```